examples/tutorial/jupyter/execution/omnisci_on_native/local/exercise_3.ipynb

```diff
@@ -0,0 +1,139 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![LOGO](../../../img/MODIN_ver2_hrz.png)\n",
+    "\n",
+    "<center><h2>Scale your pandas workflows by changing one line of code</h2>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Exercise 3: Not Implemented\n",
+    "\n",
+    "**GOAL**: Learn what happens when a function is not yet supported in Modin."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When functionality has not yet been implemented, we default to pandas\n",
+    "\n",
+    "![](../../../img/omnisci_convert_to_pandas.png)\n",
+    "\n",
+    "We convert the Modin dataframe to a pyarrow.Table, perform a lazy tree in OmniSci, render it as a pyarrow.Table, convert it to pandas to perform the operation, and then convert it back to Modin when complete. These operations will have a large overhead due to the communication involved and will take longer than pandas.\n",
+    "\n",
+    "When this is happening, a warning will be given to the user to inform them that this operation will take longer than usual. For example, `DataFrame.mask` is not yet implemented. In this case, when a user tries to use it, they will see this warning:\n",
+    "\n",
+    "```\n",
+    "UserWarning: `DataFrame.mask` defaulting to pandas implementation.\n",
+    "```"
```

Comment:
In the example below we use `DataFrame.apply` as a function that defaults to pandas (the print statements in the cells also refer to `mask`, please them as well)
```suggestion
    "When this is happening, a warning will be given to the user to inform them that this operation will take longer than usual. For example, `DataFrame.apply` is not supported. In this case, when a user tries to use it, they will see this warning:\n",
    "\n",
    "```\n",
    "UserWarning: `DataFrame.apply` defaulting to pandas implementation.\n",
    "```"
```
"is not implemented yet" -> "is not supported" because OmniSci would probably never support arbitrary python callables because of its relation algebra nature

Reply:
Let's also describe a set of limitations for OmniSci execution, the functions that will always be defaulting to pandas due to execution nature: [this ones](https://modin.readthedocs.io/en/stable/flow/modin/experimental/core/execution/native/implementations/omnisci_on_native/index.html#relational-engine-limitations) + a note about `.apply(python_callable)`

Reply:
A link to the set of limitations and a note you mentioned should be enough.